packages/@icon-magic/cli/src/index.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+
+import * as iconBuild from '@icon-magic/build';
+import * as configReader from '@icon-magic/config-reader';
+import * as iconGenerate from '@icon-magic/generate';
+import { Logger, logger } from '@icon-magic/logger';
+import * as program from 'commander';
+
+const LOGGER: Logger = logger('@icon-magic/cli/index');
+
+program
+  .command(
+    'build [inputPaths...]',
+    'construct flavors of an icon from its variants, after applying the build plugins.'
+  )
+  .action(async inputPaths => {
+    if (!inputPaths.length) {
+      LOGGER.error(
+        "No Input Directories were specified.\nDid you mean 'icon-magic build .'?"
+      );
+    }
+    // Get the iconSet from the inputPaths
+    const iconSet = configReader.getIconConfigSet(inputPaths);
+
+    // build all the icons
+    await iconBuild.build(iconSet);
+
+    // exit without any errors
+    process.exit(0);
+  });
+
+program
+  .command(
+    'generate [inputPaths...]',
+    'generates the flavors of the icon in the extension types that it can be consumed.'
+  )
+  .action(async inputPaths => {
+    if (!inputPaths.length) {
+      LOGGER.error(
+        "No Input Directories were specified.\nDid you mean 'icon-magic generate .'?"
+      );
+    }
+    // Get the iconSet from the inputPaths
+    const iconSet = configReader.getIconConfigSet(inputPaths);
+
+    // generate all the icons
+    await iconGenerate.generateFromConfigHash(iconSet);
```

Comment:
is this supposed to be `generate`? No `generateFromConfigHash` currently exists in `generate`

Reply:
Now it does :) I just pushed https://github.com/linkedinlabs/icon-magic/pull/18